config/config.go

```diff
@@ -57,6 +57,18 @@ func (c *Config) GetFilename() string {
 	return c.filename
 }
 
+func (c *Config) RunFromCron() bool {
```

Comment:
Ah hah, cunning. So it looks like we're going to record in our `config` if we expect to be running from `cron` or not...